plugins/catalog-backend/src/ingestion/index.ts

```diff
@@ -15,11 +15,6 @@
  */
 
 export type {
-  AnalyzeLocationEntityField,
-  AnalyzeLocationExistingEntity,
-  AnalyzeLocationGenerateEntity,
-  AnalyzeLocationRequest,
-  AnalyzeLocationResponse,
```

Comment:
You can actually not remove these. They are part of the public surface of the package, and the package is at a 1.x version. Removing them is a breaking change, which would force a new major version of the package to be released.

Remember, the deprecations work is more about removing _usages_ of deprecated things, rather than _exposure_ of deprecated things.

Reply:
Okay! I will change these back then.